jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -393,6 +393,8 @@ def _image_viewer_update(self, viewer, x, y):
             self.row3_unreliable = unreliable_world
             # TODO: use sky directly, but need to figure out how to have a compatible "blank" entry
             self._dict['world'] = (sky.ra.value, sky.dec.value)
```

Comment:
does it still even make sense to store world here (and same with pixel below)? Are they ever used anywhere now?